app/src/main/java/nu/yona/app/ui/LaunchActivity.java

```diff
@@ -131,7 +131,7 @@ private void ensureValidUserEntity()
 			User user = APIManager.getInstance().getAuthenticateManager().getUser();
 			URL environmentURL = new URL(YonaApplication.getEventChangeManager().getDataState().getServerUrl());
 			URL storedUserURL = new URL(user.getLinks().getSelf().getHref());
-			if (environmentURL.getProtocol() != storedUserURL.getProtocol())
+			if (!environmentURL.getProtocol().equals(storedUserURL.getProtocol()))
```

Comment:
Oops! I didn't see this one. Is it wrong in the ``master`` too?

Reply:
Yes it is in master too.

Reply:
In that case, can you make a PR on master for this?

Reply:
But I doubt if there will be any harm with it. As both the references will never be equal and app will make an extra getUser call.

Reply:
But that'll cause extra overhead to everyone. It's better to make that 8-character PR. Pushing that to production is a very small effort.

Reply:
ok.